src/ontology/mondo-edit.obo

```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
 xref: OMIM:620690 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000158 {source="OMIM:620690"} ! developmental dysplasia of the hip
 
+[Term]
+id: MONDO:0958083
+name: conjoined twins
+xref: Orphanet:647916 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019755 {source="Orphanet:647916"} ! developmental defect during embryogenesis
```

Comment:
if it is not too much trouble to re-run this, I think we started also adding our ORCID as the source, so we won't flag these later as ordo-only superclass assertions that need to be reviewed

Reply:
done :-)